integration/tests/neon_evm/test_revision.py

```diff
@@ -0,0 +1,314 @@
+        holder1 = holder_acc
+        holder2 = new_holder_acc
+        user1_revisions_before = evm_loader.get_balance_account_revision(user1.balance_account_address)
+        user2_revisions_before = evm_loader.get_balance_account_revision(user2.balance_account_address)
+        signed_tx1 = make_contract_call_trx(user1, rw_lock_contract, 'update_storage_map(uint256)',
+                                            [data_storage_acc_count])
+        write_transaction_to_holder_account(signed_tx1, holder1, operator_keypair)
+
+        emulate_result1 = neon_api_client.emulate_contract_call(user1.eth_address.hex(),
+                                                                rw_lock_contract.eth_address.hex(),
+                                                                'update_storage_map(uint256)', [data_storage_acc_count])
+        acc_from_emulation1 = [PublicKey(item['pubkey']) for item in emulate_result1['solana_accounts']]
+
+        signed_tx2 = make_contract_call_trx(user2, rw_lock_contract, 'update_storage_map(uint256)',
+                                            [data_storage_acc_count])
+
+        emulate_result2 = neon_api_client.emulate_contract_call(user2.eth_address.hex(),
+                                                                rw_lock_contract.eth_address.hex(),
+                                                                'update_storage_map(uint256)', [data_storage_acc_count])
+        acc_from_emulation2 = [PublicKey(item['pubkey']) for item in emulate_result2['solana_accounts']]
+
+        write_transaction_to_holder_account(signed_tx2, holder2, operator_keypair)
+
+        def send_transaction_steps(holder_account, accounts):
+            return send_transaction_step_from_account(operator_keypair, evm_loader, treasury_pool, holder_account,
+                                                      accounts, EVM_STEPS, operator_keypair)
+
+        send_transaction_steps(holder1, acc_from_emulation1)
+        send_transaction_steps(holder2, acc_from_emulation2)
+        send_transaction_steps(holder1, acc_from_emulation1)
+        send_transaction_steps(holder2, acc_from_emulation2)
+        resp1 = send_transaction_steps(holder1, acc_from_emulation1)
+        resp2 = send_transaction_steps(holder2, acc_from_emulation2)
+        check_transaction_logs_have_text(resp1.value.transaction.transaction.signatures[0], "exit_status=0x11")
+        check_transaction_logs_have_text(resp2.value.transaction.transaction.signatures[0], "exit_status=0x11")
+
+        check_holder_account_tag(holder1, FINALIZED_STORAGE_ACCOUNT_INFO_LAYOUT, TAG_FINALIZED_STATE)
+        check_holder_account_tag(holder2, FINALIZED_STORAGE_ACCOUNT_INFO_LAYOUT, TAG_FINALIZED_STATE)
+        user1_revision_after = evm_loader.get_balance_account_revision(user1.balance_account_address)
+        user2_revision_after = evm_loader.get_balance_account_revision(user2.balance_account_address)
+        assert user1_revisions_before == user1_revision_after
```

Comment:
why you didn't check storage revisions? And if I understand right, you change one contract cell and revision should be 2 at the end, no?

Reply:
It should be 1, because each user changes his own storage cells
I can add this check but I think it is the same as in test_call_contract_with_changing_data test

Reply:
@kristinaNikolaevaa I checked contract, agree about 1 :) but we need check this behavior toom that revision 1

Reply:
added